docker/.env.example.full

```
@@ -178,7 +178,7 @@ TOKEN_EXPIRE_TIME="604800"
 # If set, it will allow anyone to bypass verification.
 # Only use for local development or testing.
 # (Optional - handled by libraries)
-UNIVERSAL_VERIFICATION_CODE=""
+UNIVERSAL_VERIFICATION_CODE="1024"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)